src/Integration/NewConnectedMode/IConfigurationPersister.cs

```diff
@@ -54,44 +55,26 @@ public BindingConfiguration Persist(BoundSonarQubeProject project, SonarLintMode
                 throw new ArgumentNullException(nameof(project));
             }
 
-            var targetDirectory = GetTargetDirectory(bindingMode);
```

Comment:
The old `GetTargetDirectory` was inaccurately named - it was actually returning a file path, so I renamed it.